CONTRIBUTING.md

```diff
@@ -66,3 +70,9 @@ git add -A
 git commit -sm "one commit on <branch-name>"
 git push --force
 ```
+
+### Code of Conduct
+
+This project has a [Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md), which everybody involved must follow. Please take a moment to familiarize yourself with this document.
```

Comment:
Let's point to our code of conduct instead, in case we extend it.
```suggestion
This project has a [Code of Conduct](https://github.com/kedacore/.github/blob/main/CODE_OF_CONDUCT.md), which everybody involved must follow. Please take a moment to familiarize yourself with this document.
```

Reply:
@tomkerkhove done